### ISSUE TITLE
Runs on simulator but not on a device

### ISSUE THREAD

**Issue by fmatosic** [twilio/video-quickstart-ios#492]
> Before filing an issue please check that the issue is not already addressed by the following:
>
>  * [Video Guides](https://www.twilio.com/docs/api/video)
>  * [Changelog](https://www.twilio.com/docs/api/video/changelog-twilio-video-ios-version-2x).
> 
> If this is an issue with the QuickStart itself, file it here. If this is an issue with the SDK or how to do something with the SDK please use [twilio-video-ios](https://github.com/twilio/twilio-video-ios/issues) instead.
>
> Please ensure that you are not sharing any
[Personally Identifiable Information(PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii)
or sensitive account information (API keys, credentials, etc.) when reporting an issue.

### Description

Running on a iphone xs max results in an error but on a simulator works fine.

### Steps to Reproduce

just use a device instead of a simulator

#### Code

```swift
// Code that helps reproduce the issue
```

#### Expected Behavior

To run same as on simulator

#### Actual Behavior

just crashes

#### Reproduces How Often

[What percentage of the time does it reproduce?]

#### Logs
Debug level logs are helpful when investigating issues. To enable debug level logging, add the following code to your application:

```.swift
TwilioVideoSDK.setLogLevel(.debug)

// Log output when the issue occurs
dyld: Library not loaded: @rpath/TwilioVideo.framework/TwilioVideo
  Referenced from: /private/var/containers/Bundle/Application/330620BF-99D8-411E-90EF-73122571D13B/VideoQuickStart.app/VideoQuickStart
  Reason: no suitable image found.  Did find:
	/private/var/containers/Bundle/Application/330620BF-99D8-411E-90EF-73122571D13B/VideoQuickStart.app/Frameworks/TwilioVideo.framework/TwilioVideo: code signature invalid for '/private/var/containers/Bundle/Application/330620BF-99D8-411E-90EF-73122571D13B/VideoQuickStart.app/Frameworks/TwilioVideo.framework/TwilioVideo'

	/private/var/containers/Bundle/Application/330620BF-99D8-411E-90EF-73122571D13B/VideoQuickStart.app/Frameworks/TwilioVideo.framework/TwilioVideo: code signature invalid for '/private/var/containers/Bundle/Application/330620BF-99D8-411E-90EF-73122571D13B/VideoQuickStart.app/Frameworks/TwilioVideo.framework/TwilioVideo'

	/private/var/containers/Bundle/Application/330620BF-99D8-411E-90EF-73122571D13B/VideoQuickStart.app/Frameworks/TwilioVideo.framework/TwilioVideo: stat() failed with errno=1
	/private/var/containers/Bundle/Application/330620BF-99D8-411E-90EF-73122571D13B/VideoQuickStart.app/Frameworks/TwilioVideo.framework/TwilioVideo: code signature invalid for '/private/var/containers/Bundle/Application/330620BF-99D8-411E-90EF-73122571D13B/VideoQuickStart.app/Frameworks/TwilioVideo.framework/TwilioVideo'

	/private/var/containers/Bundle/Application/330620BF-99D8-411E-90EF-73122571D13B/VideoQuickStart.app/Frameworks/TwilioVideo.framework/TwilioVideo: stat() failed with errno=1
(lldb) 


### Versions

 pod 'TwilioVideo', '~> 3.2'

#### Video iOS SDK

[e.g. 1.3.12 via CocoaPods]

#### Xcode

xcode version: Version 11.4.1 (11E503a)
#### iOS Version

13.3.1

#### iOS Device

XS max

**Comment by paynerc** [comment]
@fmatosic,

Are you using a free Apple Developer Account for signing the application? The issue you are reporting here sounds very similar to #460. This is an iOS issue and is resolved by upgrading your device to iOS 13.4.1.

Ryan

**Comment by fmatosic** [comment]
Seems when I lowered the target to ios 11.0 issue is gone and it starts up normal
